core/primitives-core/src/config.rs

```diff
@@ -165,6 +166,13 @@ impl Default for VMLimitConfig {
     }
 }
 
+/// Configuration of view methods execution, during which no costs should be charged.
+#[derive(Default, Clone, Serialize, Deserialize, Debug, Hash, PartialEq, Eq)]
+pub struct ViewConfig {
+    /// If specified, defines max burnt gas per view method.
+    pub max_gas_burnt_view: Gas,
```

Comment:
```suggestion
    pub max_gas_burnt: Gas,
```

"If sepecified" bit is wrong -- this is no longer an option. The `_vew` suffix is reduntant, we are in a `ViewConfig` (rule-of-thumb -- each part of a compound name should add info). The comment somewhat repeats the name,  and feels like it can be removed.